clients/Android/ultrablue/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="fr.gouv.ssi.ultrablue">
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
```

Comment:
Add BLUETOOTH_SCAN: https://developer.android.com/guide/topics/connectivity/bluetooth/permissions

Reply:
See, that's why you should resolve review comments ;-)